projects/lido/index.js

```diff
@@ -58,6 +58,21 @@ async function ksm(timestamp, ethBlock, chainBlocks) {
   }
 }
 
+async function dot(timestamp, ethBlock, chainBlocks) {
+  const chain = "moonbeam"
+  const block = await getBlock(timestamp, chain, chainBlocks, true)
+  const pooledCoin = await sdk.api.abi.call({
+    block,
+    chain,
+    target: "0xFA36Fe1dA08C89eC72Ea1F0143a35bFd5DAea108",
```

Comment:
Convert address to lower case for consistency with other addresses

Reply:
> Convert address to lower case for consistency with other addresses
Done